smoketests/tests/permissions.py

```diff
@@ -10,8 +10,6 @@ def setUp(self):
     def test_call(self):
         """Ensure that anyone has the permission to call any standard reducer"""
 
-        self.new_identity()
```

Comment:
I don't believe this `new_identity` really added anything to the test, but it hamstrung the test when running against servers that required a `spacetime login`.